plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/core/project/PathUtil.java

```diff
@@ -0,0 +1,157 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.maven.server.core.project;
+
+import org.apache.commons.io.output.NullWriter;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.function.Supplier;
+
+/**
+ * Set of util methods for path strings.
+ */
+public class PathUtil {
```

Comment:
I think that [che-core-commons-lang](https://github.com/eclipse/che/tree/master/core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/os) is better place for this util class.
Especially since we already have a special package **org.eclipse.che.commons.lang.os** there for the platform-specific utils.